templates/geomapshark/index.html

```diff
@@ -9,19 +9,42 @@
 {% endblock %}
 
 
-{% block sidebar %}
-    <ul class="sidebar-nav app-list" style="margin-top: 50px; margin-left: 5%; width: 300px;">
-        {% if user.groups.all.0.name == "extern" %}
 
-          <li><a class="btn btn-primary responsive-btn" href="{% url 'gpf:listexterns' %}"><i class="fa fa-road"></i>&nbsp;Permis de fouille</a></li>
-          <li><a class="btn btn-primary responsive-btn" href="{{apps.mapnv}}" target="_blank"><i class="fa fa-map"></i>&nbsp;mapnv: Géoportail</a></li>
-          <li><a class="btn btn-primary responsive-btn" href="/lidar" target="_blank"><i class="fa fa-cube"></i>&nbsp;mapn: Géoportail 3D (LiDAR)</a></li>
+{% block content %}
+
+
+
+        {% if user.groups.all.0.name == "extern" %}
```

Comment:
Out of scope (since this code was already here), but still: this is fragile because if the user has more than 1 group, there's no guarantee the `extern` group will be in first position. You'd better use a custom template tag to render this. You'll then be able to execute some specific code to check if the user is in the `extern` group (eg. `request.user.groups.filter(name='extern').exists()`).

Strictly Django speaking though, you should use [permissions](https://docs.djangoproject.com/en/3.0/ref/contrib/auth/#django.contrib.auth.models.User.has_perm) instead of checking if a user is in a specific group.